src/main/java/com/icthh/xm/tmf/ms/resource/config/AppMetricsConfiguration.java

```diff
@@ -4,12 +4,13 @@
 import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
 import com.ryantenney.metrics.spring.config.annotation.MetricsConfigurerAdapter;
 import com.zaxxer.hikari.HikariDataSource;
-import javax.annotation.PostConstruct;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Configuration;
 
+import javax.annotation.PostConstruct;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version